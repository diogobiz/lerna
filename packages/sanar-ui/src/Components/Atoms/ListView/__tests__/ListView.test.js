import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESListView from '../ListView'
import ESListViewItem from '../ListViewItem'

it('renders correctly', () => {
    const component = (
        <ESListView>
            <ESListViewItem>>Here is some content!</ESListViewItem>
        </ESListView>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
