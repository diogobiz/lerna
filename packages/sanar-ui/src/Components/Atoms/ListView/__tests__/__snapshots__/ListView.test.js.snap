// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-list ant-list-split"
>
  <div
    className="ant-spin-nested-loading"
  >
    <div
      className="ant-spin-container"
    >
      <li
        className="ant-list-item es-list-view-item"
      >
        &gt;Here is some content!
      </li>
    </div>
  </div>
</div>
`;
